### PR TITLE
failure modes

### PR DESCRIPTION
**Pull request by chrisantonellis**
## Description
* wrote some content for `failure_modes` doc
* fixes https://github.com/wayfair-incubator/auxeng-docs/issues/18

## Type of Change

- [ ] Bug Fix
- [ ] New Feature
- [ ] Breaking Change
- [ ] Refactor
- [X] Documentation
- [ ] Other (please describe)

## Checklist

- [X] I have read the [contributing guidelines](https://github.com/wayfair-incubator/auxeng-docs/blob/main/CONTRIBUTING.md)
- [X] Existing issues have been referenced (where applicable)
- [X] I have verified this change is not present in other open pull requests
- [X] Functionality is documented
- [ ] All code style checks pass
- [X] New code contribution is covered by automated tests
- [X] All new and existing tests pass

### CODE REVIEW

File: src/docs/theory/failure_modes.md [hunk 1]
```diff
@@ -3,16 +3,53 @@ title: "Failure Modes"
...
-- AuxEng feeling like an outsider
-- Lack of improvements to Platform layer
-- Lack of psychological safety in retros
-  - Welcome and reward critical feedback
-- Being just "embedded engineering"
-- Bad ratio of AuxEng to team members (1to6 is upper bound)
+TODO:
+* how to identify, avoid these situations
+
+An auxiliary engagement is a hopeful endeavor; there is no guarantee of success.
```
Comment (GaryPWhite) [inline src/docs/theory/failure_modes.md:9]:
```suggestion
An Auxiliary Engagement is a hopeful endeavor; there is no guarantee of success.
```

capital letters for proper nouns I think

File: src/docs/theory/failure_modes.md [hunk 2]
```diff
@@ -3,16 +3,53 @@ title: "Failure Modes"
...
+
+1. **A disconnect between the team and leadership**  
+    If there is a misunderstanding or misalignment between the team and
+    leadership, this can undermine the stability of the engagement. This could
+    take the form of excited leadership with under-engaged team, or a team with
+    one goal in mind and leadership with another. This can result in a team that
+    commits to a large idea on paper but does not execute, and cancelled
+    partnerships mid-engagement.
+
+1. **Lack of improvements to Platform layer**
```
Comment (GaryPWhite) [inline src/docs/theory/failure_modes.md:37]:
```suggestion
1. **Lack of improvements to platform layer**  
```

given that later and elsewhere you mention the Platfrom as a (p)latform, I think it should be lowercase here!

**Review by GaryPWhite — APPROVED** [review]
super concise and helpful information / context. Thanks for it!

**Comment by chrisantonellis** [comment]
fixing this up now!

**Pull request merged**